packages/validation/src/validators/practices/rules/KBP116-image-tagged.ts

```diff
@@ -0,0 +1,26 @@
+import {defineRule} from '../../custom/config.js';
+import {validatePodSpec} from '../../custom/utils.js';
+
+export const imageTagged = defineRule({
+  id: 116,
+  description: 'Require images to be tagged.',
+  fullDescription:
+    'Untagged images will pull undetermined image versions. Adding explicit tags makes it easy to understand what is deployed and helps with rollbacks in case of problems.',
+  help: 'Add a tag to the image.',
+  advanced: {
+    enabled: false,
+    severity: 5,
+  },
+  validate({resources}, {report}) {
+    validatePodSpec(resources, (resource, pod, prefix) => {
+      pod.containers.forEach((container, i) => {
+        const parts = container.image?.split(':');
+        const valid = parts?.at(0) !== undefined;
```

Comment:
Shouldn't it be `parts?.at(1) !== undefined;`? Because you will always get value at 0 by splinting by non-existent token, like:

```js
'panda-sidecar:latest'.split(':')
// ['panda-sidecar', 'latest']
'panda-sidecar'.split(':')
// ['panda-sidecar']
```